website/docs/configure_hydra/Intro.md

```diff
@@ -90,4 +90,4 @@ minor_version: ${python_version:minor}                     # runtime python vers
 micro_version: ${python_version:micro}                     # runtime python version in the format major.minor.micro, eg: 3.8.2
 ```
 
-You can learn more about OmegaConf <a class="external" href="https://omegaconf.readthedocs.io/en/latest/usage.html#access-and-manipulation" target="_blank">here</a>.
\ No newline at end of file
+You can learn more about OmegaConf <a class="external" href="https://omegaconf.readthedocs.io/en/latest/usage.html#access-and-manipulation" target="_blank">here</a>.
```

Comment:
curious, why is this showing as a diff?

Reply:
I used the github's "commit this change" thingy, seems like something wrong with that